modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.client/src/main/scala/edu/gemini/seqexec/web/client/services/SeqexecWebClient.scala

```diff
@@ -61,7 +61,7 @@ object SeqexecWebClient extends ModelBooPicklers {
     */
   def stop(s: SequenceView): Future[RegularCommand] = {
     Ajax.post(
-      url = s"$baseUrl/commands/${s.id}/stop",
+      url = s"$baseUrl/commands/${s.id}/pause",
```

Comment:
The other route it's from the previous implementation and didn't remove it because I don't know if we'll need it later. I'll comment it out.